07-multi-hop/07-multi-hop.md

```diff
@@ -1,12 +1,38 @@
 ## Goal: Check multi-hop connectivity over IPv6
 
+### Pre-setup Steps
+1. Book n>15 iotlab M3 nodes (preferably in grenoble) in a straight line 3 nodes apart. (so x--x--x--x)
+2. Add the following CFLAGS to the `tests/gnrc_udp`:
+```
+CFLAGS += -DGNRC_PKTBUF_SIZE=512 -DGNRC_NETIF_IPV6_ADDRS_NUMOF=20 \
```

Comment:
Why do you make the packet buffer smaller here?

Reply:
I just took the flags from minimal makefile.  TBH I don't understand everything that goes on., this is just what happened to work (although a lot of instruction was given to me by @cgundogan).  Shall I remove it?

Reply:
I'd argue to keep it as default as possible. Especially for forwarding nodes using `gnrc_minimal` as a template doesn't make much sense, as `gnrc_minimal` is in host configuration and what you need for forwarding is router configuration.

Reply:
But if you got the instructions from @cgundogan I'd like to get his input first.

Reply:
OK, I can retest with defaults to see if everything works (after all comments are addressed though)

Reply:
You might get problems with RPL and **15** nodes ;-).

Reply:
But `GNRC_NETIF_IPV6_ADDRS_NUMOF=20` really isn't required. You just need 2 (global and link-local address) per node.

Reply:
Here's the doc for the macros you are configuring here. Based on that you should be able to make a informed decision on the configuration yourself:

- [`GNRC_PKTBUF_SIZE`](http://doc.riot-os.org/group__net__gnrc__pktbuf.html#ga83a14b0588bb8adc93b565c4ebc5ec56)
- [`GNRC_NETIF_IPV6_ADDRS_NUMOF`](http://doc.riot-os.org/netif_2conf_8h.html#a0d2713931d504ecd972747894c3b08ef)
- [`GNRC_IPV6_NIB_NUMOF`](http://doc.riot-os.org/group__net__gnrc__ipv6__nib__conf.html#gad549499055dcce8399ebf30f6cb5073c)
- [`GNRC_IPV6_NIB_OFFL_NUMOF`](http://doc.riot-os.org/group__net__gnrc__ipv6__nib__conf.html#gae319bc32b0670b25529b746b9aa019b1)